.drone.yml

```diff
@@ -119,6 +119,14 @@ steps:
   - codespell
   - shellcheck
 
+- name: lighthouse
+  image: grafana/build-container:1.2.28
+  commands:
+  - lhci autorun || echo "LHCI failed!"
```

Comment:
I think this is best moved to the test-frontend step, don't see any reason to not run it as part of frontend testing?

Reply:
Would that mean testing with a dev build? Just asking because lighthouse makes most sense when run against a production build

Reply:
It should work the same as putting it in another step. Basically, Drone will build Grafana front-end code in the build-frontend step, and the test-frontend step will have the same checkout and build artifacts in its working directory (since Drone shares the working directory between the pipeline steps).

Reply:
Maybe I'm reading drone.yml wrong, but doesn't build-frontend come after test-frontend?

Reply:
@kaydelaney Yes you're correct. Do you mean that lighthouse has to run after building? In that case you can just insert it into the `commands` list of the `build-frontend` step, right?

Does lighthouse take long BTW?